examples/test/pool/components/spaceship.js

```diff
@@ -0,0 +1,53 @@
+/* global AFRAME */
+
+AFRAME.registerComponent('spaceship', {
+  init: function () {
+    var el = this.el;
+    var geometry = 'primitive: box; height: 2; width: 2; depth: 2;';
+    var material = 'color: #167341; roughness: 1.0; metalness: 0.2;';
+    el.setAttribute('geometry', geometry);
+    el.setAttribute('material', material);
+    this.onKeyDown = this.onKeyDown.bind(this);
+  },
+
+  play: function () {
+    this.attachKeyEventListeners();
+  },
+
+  pause: function () {
+    this.removeKeyEventListeners();
+  },
+
+  onKeyDown: function (evt) {
+    var el = this.el;
+    var laserEl;
+    var position;
+    if (evt.keyCode !== 32) { return; }
+    laserEl = el.sceneEl.components.pool.requestEntity();
```

Comment:
aps sorry I didn't notice that `pool` was the name of the component and not a global that will store all the pools. I mistook this use case with accessing directly the system.